boards/shields/atmel_oled1_xpro/Kconfig.shield

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 The Zephyr Project Contributors
```

Comment:
@kartben, Is this copyright valid in current context of individual contribution ?

Reply:
This form is generally not used in the Zephyr project.